source/db/_tests/__snapshots__/_json.test.ts.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Raises with a bad URL 1`] = `[SyntaxError: Unexpected token u in JSON at position 0]`;
+exports[`Raises with a bad URL 1`] = `[Error: Could not find find a JSON file for Peril settings. It's likely that Peril cannot connect to orta/other@settings.json, check the logs for more info above here. You'll probably need to make changes to your  "DATABASE_JSON_FILE" in your ENV vars.]`;
```

Comment:
@orta I had to update this to pass the tests because of the error message changing, but I couldn't figure out why the error message changed with my changes. I _think_ the behaviour is correct, so I've updated it, but if you could have a check that would be great.

Reply:
Looking a the test

```
it("Raises with a bad URL", async () => {
  mockGHContents.mockImplementationOnce(() => Promise.resolve(""))
  expect.assertions(1)

  try {
    const db = jsonDB("orta/other@settings.json")
    await db.setup()
  } catch (error) {
    expect(error).toMatchSnapshot()
  }
})
```

That feels like the actual response it's looking for, maybe this test wasn't quite right until this PR